pkg/varlinkapi/virtwriter/virtwriter.go

```diff
@@ -161,7 +167,16 @@ func Reader(r *bufio.Reader, output, errput, input io.Writer, resize chan remote
 				execEcChan <- int(ecInt)
 			}
 			return nil
-
+		case HangUpFromClient:
+			// This sleep allows the pipes to flush themselves before tearing everything down.
```

Comment:
Are you doing it?  Seems like you have commented it out?

Reply:
fixed